### PR TITLE
Add more details to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by perotinus**
/cc @pmorie, @madhusudancs

@font

/sig multicluster

### CODE REVIEW

**Comment by perotinus** [comment]
Addresses #101.

**Comment by perotinus** [comment]
@pmorie @madhusudancs @font Any issues with this?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,37 @@
 # Contributing guidelines
 
-This project is in its very early stages and is not actively soliciting
-contributions at the present time.
+If you're looking for something to work on, please look at our 
+[milestones](https://github.com/kubernetes/cluster-registry/milestones) or our
+[open issues](https://github.com/kubernetes/cluster-registry/issues)
+and assign an issue to yourself.
```
Comment (pmorie) [inline CONTRIBUTING.md]: Self-assign will only be supported for people that are already in the kubernetes org (silly limitation of github IMO).  Should we keep this language?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,37 @@
...
+    then you'll need to sign a
+    [corporate CLA](https://identity.linuxfoundation.org/node/285/organization-signup).
+
+Follow either of the two links above to access the appropriate CLA and
+instructions for how to sign and return it. Once we receive it, we'll be able
+to accept your pull requests.
+
+### Getting started with development
+
+Please refer to the [developer documentation](/docs/development.md) to get
```
Comment (pmorie) [inline CONTRIBUTING.md]: nit: the link says 'developer' but this is called 'development' everywhere else.

File: CONTRIBUTING.md (hunk 1 above)
Comment (perotinus) [reply CONTRIBUTING.md]: I've added some more details and a link to the k8s community membership docs. WDYT?

File: CONTRIBUTING.md (hunk 2 above)
Comment (perotinus) [reply CONTRIBUTING.md]: Good eye, fixed.

**Pull request merged**